diff --git a/node_modules/@formatjs/ecma402-abstract/types/date-time.d.ts b/node_modules/@formatjs/ecma402-abstract/types/date-time.d.ts
index 115d034..fdada9c 100755
--- a/node_modules/@formatjs/ecma402-abstract/types/date-time.d.ts
+++ b/node_modules/@formatjs/ecma402-abstract/types/date-time.d.ts
@@ -110,7 +110,7 @@ export interface DateTimeFormatLocaleInternalData {
 export declare type IntervalFormatsData = {
     intervalFormatFallback: string;
 } & Record<string, Record<string, string>>;
-export interface DateTimeFormat extends Omit<Intl.DateTimeFormat, 'resolvedOptions'> {
+export interface DateTimeFormat extends Omit<Intl.DateTimeFormat, 'resolvedOptions' | 'formatRangeToParts'> {
     resolvedOptions(): ResolvedDateTimeFormatOptions;
     formatRange(startDate: number | Date, endDate: number | Date): string;
     formatRangeToParts(startDate: number | Date, endDate: number | Date): IntlDateTimeFormatPart[];
