diff --git a/node_modules/@intlify/vite-plugin-vue-i18n/lib/index.cjs b/node_modules/@intlify/vite-plugin-vue-i18n/lib/index.cjs
index 2ae7007..809aa0a 100644
--- a/node_modules/@intlify/vite-plugin-vue-i18n/lib/index.cjs
+++ b/node_modules/@intlify/vite-plugin-vue-i18n/lib/index.cjs
@@ -88,14 +88,14 @@ function pluginI18n(options = { forceStringify: false }) {
         if (shared.isArray(config.resolve.alias)) {
           config.resolve.alias.push({
             find: getAliasName(),
-            replacement: `${installedPkg}/dist/${installedPkg}.runtime.esm-bundler.js`
+            replacement: `${installedPkg}/dist/${installedPkg}.runtime.esm-bundler.mjs`
           });
         } else {
-          config.resolve.alias[getAliasName()] = `${installedPkg}/dist/${installedPkg}.runtime.esm-bundler.js`;
+          config.resolve.alias[getAliasName()] = `${installedPkg}/dist/${installedPkg}.runtime.esm-bundler.mjs`;
         }
         debug(`alias name: ${getAliasName()}`);
         debug(
-          `set ${installedPkg} runtime only: ${installedPkg}/dist/${installedPkg}.runtime.esm-bundler.js`
+          `set ${installedPkg} runtime only: ${installedPkg}/dist/${installedPkg}.runtime.esm-bundler.mjs`
         );
       } else if (command === "serve" && installedPkg === "petite-vue-i18n" && useVueI18nImportName) {
         normalizeConfigResolveAlias(config);
diff --git a/node_modules/@intlify/vite-plugin-vue-i18n/lib/index.mjs b/node_modules/@intlify/vite-plugin-vue-i18n/lib/index.mjs
index 2b5499c..0c066fc 100644
--- a/node_modules/@intlify/vite-plugin-vue-i18n/lib/index.mjs
+++ b/node_modules/@intlify/vite-plugin-vue-i18n/lib/index.mjs
@@ -79,14 +79,14 @@ function pluginI18n(options = { forceStringify: false }) {
         if (isArray(config.resolve.alias)) {
           config.resolve.alias.push({
             find: getAliasName(),
-            replacement: `${installedPkg}/dist/${installedPkg}.runtime.esm-bundler.js`
+            replacement: `${installedPkg}/dist/${installedPkg}.runtime.esm-bundler.mjs`
           });
         } else {
-          config.resolve.alias[getAliasName()] = `${installedPkg}/dist/${installedPkg}.runtime.esm-bundler.js`;
+          config.resolve.alias[getAliasName()] = `${installedPkg}/dist/${installedPkg}.runtime.esm-bundler.mjs`;
         }
         debug(`alias name: ${getAliasName()}`);
         debug(
-          `set ${installedPkg} runtime only: ${installedPkg}/dist/${installedPkg}.runtime.esm-bundler.js`
+          `set ${installedPkg} runtime only: ${installedPkg}/dist/${installedPkg}.runtime.esm-bundler.mjs`
         );
       } else if (command === "serve" && installedPkg === "petite-vue-i18n" && useVueI18nImportName) {
         normalizeConfigResolveAlias(config);
