diff --git a/node_modules/@vitejs/plugin-vue-jsx/index.js b/node_modules/@vitejs/plugin-vue-jsx/index.js
index 8e7898a..4aff157 100644
--- a/node_modules/@vitejs/plugin-vue-jsx/index.js
+++ b/node_modules/@vitejs/plugin-vue-jsx/index.js
@@ -191,16 +191,16 @@ function vueJsxPlugin(options = {}) {
         }
 
         if (hotComponents.length) {
+          if (hasDefault && (needHmr || ssr)) {
+            result.code =
+              result.code.replace(
+                /export default defineComponent/g,
+                `const __default__ = defineComponent`
+              ) + `\nexport default __default__`
+          }
+
           if (needHmr && !ssr) {
             let code = result.code
-            if (hasDefault) {
-              code =
-                code.replace(
-                  /export default defineComponent/g,
-                  `const __default__ = defineComponent`
-                ) + `\nexport default __default__`
-            }
-
             let callbackCode = ``
             for (const { local, exported, id } of hotComponents) {
               code +=
