diff --git a/dist/client/__vitest_browser__/orchestrator-jtzFEKPy.js b/dist/client/__vitest_browser__/orchestrator-jtzFEKPy.js
index 0000000000000000000000000000000000000000..0000000000000000000000000000000000000000 100644
--- a/dist/client/__vitest_browser__/orchestrator-jtzFEKPy.js
+++ b/dist/client/__vitest_browser__/orchestrator-jtzFEKPy.js
@@ -395,18 +395,9 @@ async function setIframeViewport(iframe, width, height) {
       [{ width, height }]
     );
   } else {
-    const scale = Math.min(
-      1,
-      iframe.parentElement.parentElement.clientWidth / width,
-      iframe.parentElement.parentElement.clientHeight / height
-    );
-    iframe.parentElement.style.cssText = `
-      width: ${width}px;
-      height: ${height}px;
-      transform: scale(${scale});
-      transform-origin: left top;
-    `;
-    (_b = iframe.parentElement) == null ? void 0 : _b.setAttribute("data-scale", String(scale));
+    iframe.style.width = `${width}px`
+    iframe.style.height = `${height}px`
+    iframe.parentElement?.setAttribute('data-scale', '1')
     await new Promise((r) => requestAnimationFrame(r));
   }
 }
