diff --git a/dist/chunk-YOHL3P33.js b/dist/chunk-YOHL3P33.js
index b842e4b21264a2e215c5ff82be21fdf90285df62..ba7fef15a435d6512b59067bdf187fdab7b44170 100644
--- a/dist/chunk-YOHL3P33.js
+++ b/dist/chunk-YOHL3P33.js
@@ -584,6 +584,7 @@ var Context = class {
     this._componentNameMap = {};
     Array.from(this._componentPaths).forEach((path) => {
       const name = pascalCase(getNameFromFilePath(path, this.options));
+      if (this.options.excludeNames?.includes(name)) return;
       if (this._componentNameMap[name] && !this.options.allowOverrides) {
         console.warn(`[unplugin-vue-components] component "${name}"(${path}) has naming conflicts with other components, ignored.`);
         return;
